controller/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "dis-spawner-controller"
```

Comment:
Just curious: why the name `controller` instead of `scheduler`?

Reply:
My thought was that both would live in the same "controller" binary, similar to how the proxy, agent, and cert-refresher live in the "drone" binary. I'm not sure yet whether it should be possible to run both in the same process (which simplifies local test deployment) or not, but even if not, we could build them into the same container and switch between them as a runtime flag.

I'm also tempted to combine the drone and controller into one binary so that we just have one image that does everything, and then we run it with different flags to get the different components.